### PR TITLE
Add Calculator component

### PR DESCRIPTION
**Pull request by Berkay-akbas**
## Changes
- Deleted all the boilerplate from CRA.
- Created the components directory.
- Calculator.js file created in components and  UI of the calculator added.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (ShoiraTa) [inline src/App.js:7]: - [ ] The `App.js` component should be class-based. Kindly refactor it 👍🏻

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @Berkay-akbas ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

## Let me point out ⭐ 
- Amazing job implementing the UI of the calculator, it looks awesome! 👍🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## Optional suggestions

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
-  return (
-    <div className="App">
-      <Calculator />
-    </div>
-  );
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.name = 'eslint';
+  }
```
Comment (muneebulrehman) [inline src/App.js:9]: - Optional
Since there is no use of constructor in this code, you can consider removing this portion from your component.

**Review by muneebulrehman — APPROVED** [review]
Hi @Berkay-akbas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**